packages/app/src/components/ModeSelector.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <div class="container centered-container" justify-around style="margin-top: 20px;">
+      <div style="max-width:140px;" class="container column-container centered-container">
+        <q-btn
+          v-if="isTimedMode"
+          @click="setTimedMode"
+          round
+          size="40px"
+          color="primary"
+          icon="alarm"
+        />
+        <q-btn v-else @click="setTimedMode" round size="40px" icon="alarm" />
+        <p style="text-align:center;">How long would you like to play for?</p>
```

Comment:
Use Quasar's helper classes for text alignment as well https://quasar.dev/style/typography#CSS-Helper-Classes